lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
+                rule.check(file, errors);
+            } catch (e) {
+                errors.add(e, 1, 0);
```

Comment:
> Typed errors can help us to have it with Errors object.

Not sure if i understand you, why exactly we need to pass it to another function?